src/public/stylesheets/application.min.css

```diff
@@ -1 +1 @@
ght-5{padding-right:25px!important}.govuk-\!-padding-bottom-5{padding-bottom:25px!important}.govuk-\!-padding-left-5{padding-left:25px!important}}.govuk-\!-padding-6{padding:20px!important}@media (min-width:40.0625em){.govuk-\!-padding-6{padding:30px!important}}.govuk-\!-padding-top-6{padding-top:20px!important}.govuk-\!-padding-right-6{padding-right:20px!important}.govuk-\!-padding-bottom-6{padding-bottom:20px!important}.govuk-\!-padding-left-6{padding-left:20px!important}@media (min-width:40.0625em){.govuk-\!-padding-top-6{padding-top:30px!important}.govuk-\!-padding-right-6{padding-right:30px!important}.govuk-\!-padding-bottom-6{padding-bottom:30px!important}.govuk-\!-padding-left-6{padding-left:30px!important}}.govuk-\!-padding-7{padding:25px!important}@media (min-width:40.0625em){.govuk-\!-padding-7{padding:40px!important}}.govuk-\!-padding-top-7{padding-top:25px!important}.govuk-\!-padding-right-7{padding-right:25px!important}.govuk-\!-padding-bottom-7{padding-bottom:25px!important}.govuk-\!-padding-left-7{padding-left:25px!important}@media (min-width:40.0625em){.govuk-\!-padding-top-7{padding-top:40px!important}.govuk-\!-padding-right-7{padding-right:40px!important}.govuk-\!-padding-bottom-7{padding-bottom:40px!important}.govuk-\!-padding-left-7{padding-left:40px!important}}.govuk-\!-padding-8{padding:30px!important}@media (min-width:40.0625em){.govuk-\!-padding-8{padding:50px!important}}.govuk-\!-padding-top-8{padding-top:30px!important}.govuk-\!-padding-right-8{padding-right:30px!important}.govuk-\!-padding-bottom-8{padding-bottom:30px!important}.govuk-\!-padding-left-8{padding-left:30px!important}@media (min-width:40.0625em){.govuk-\!-padding-top-8{padding-top:50px!important}.govuk-\!-padding-right-8{padding-right:50px!important}.govuk-\!-padding-bottom-8{padding-bottom:50px!important}.govuk-\!-padding-left-8{padding-left:50px!important}}.govuk-\!-padding-9{padding:40px!important}@media (min-width:40.0625em){.govuk-\!-padding-9{padding:60px!important}}.govuk-\!-padding-top-9{padding-top:40px!important}.govuk-\!-padding-right-9{padding-right:40px!important}.govuk-\!-padding-bottom-9{padding-bottom:40px!important}.govuk-\!-padding-left-9{padding-left:40px!important}.govuk-\!-font-size-80{font-size:53px!important;font-size:3.3125rem!important;line-height:1.03774!important}@media (min-width:40.0625em){.govuk-\!-padding-top-9{padding-top:60px!important}.govuk-\!-padding-right-9{padding-right:60px!important}.govuk-\!-padding-bottom-9{padding-bottom:60px!important}.govuk-\!-padding-left-9{padding-left:60px!important}.govuk-\!-font-size-80{font-size:80px!important;font-size:5rem!important;line-height:1!important}}@media print{.govuk-warning-text__icon{font-family:sans-serif}.govuk-\!-font-size-80{font-size:53pt!important;line-height:1.1!important}}.govuk-\!-font-size-48{font-size:32px!important;font-size:2rem!important;line-height:1.09375!important}@media (min-width:40.0625em){.govuk-\!-font-size-48{font-size:48px!important;font-size:3rem!impor
```

Comment:
This is the minified css, so difficult to tell what has been changed, essentially, the same as the scss file, adding a max-width:78ex